sdk/go/common/util/logging/log.go

```diff
@@ -43,8 +43,28 @@ var LogFlow = false     // true to flow logging settings to child processes.
 var rwLock sync.RWMutex
 var filters []Filter
 
-func V(level glog.Level) glog.Verbose {
-	return glog.V(level)
+type VerboseLogger glog.Verbose
```

Comment:
Consider adding a comment to this.

```suggestion
// VerboseLogger logs messages only if verbosity matches the level it was built with.
//
// It may be used as a boolean to check if it's enabled.
//
//	if log := logging.V(lvl); log {
//		log.Infoln(expensiveComputation())
//	}
type VerboseLogger glog.Verbose
```